scripts/gee

```diff
@@ -4910,6 +4910,65 @@ function gee__bazelgc() {
   fi
 }
 
+##########################################################################
+# bisect command
+##########################################################################
+
+_register_help "bisect" "Find a commit that caused a command to fail." <<'EOT'
+Usage: gee bisect [command...]
+
+This command wraps the `git bisect` command, and attempts to discover
+a commit that causes the provided command to transition from a success
+to a failure.  During this process, gee will create a special branch
+named `bisect_<branchname>` to perform the bisect operation in.
+
+gee will attempt to find a previous last good commit by testing a day, a
+week, a month, 3 months, and finally six months into the past.  Once
+a past good commit is found, the `git bisect` command will be used to
+identify the commit that caused a transition from a pass to a fail.
+
+gee assumes that the provided command will fail at the head revision.
+If this is not the case, the behavior of this command is undefined.
+
+EOT
+
+function gee__bisect() {
+  local CMD
+  CMD=( "$@" )
+  local CUR_BRANCH BISECT_BRANCH
+  CUR_BRANCH="$(_get_current_branch)"
+  BISECT_BRANCH="BISECT_${CUR_BRANCH}"
+  gee__mkbr "${BISECT_BRANCH}"
+  _checkout_or_die "${BISECT_BRANCH}"
+  _git reset --hard "${CUR_BRANCH}"
```

Comment:
This will erase any local changes. Is that OK?

Reply:
It will erase any local changes in the newly created bisect branch.  I should probably make sure a branch with that name doesn't already exist, but it won't change the user's original branch.